examples/kubernetes/op-connect-deployment.yaml

```diff
@@ -31,13 +30,14 @@ spec:
               name: shared-data
       containers:
         - name: connect-api
-          image: 1password/connect-api:latest
+          image: 1password/connect-api:1.5.7
```

Comment:
Why do we want to use here exact version?
That would require updating it when we rollout the new Connect version.

Reply:
Thanks for taking a look! This is a Kubernetes best practice to help accurately track the version image and roll back in case of issues (see the `Note:` block in the [Image pull policy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) section of the [Images](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) page in the Kubernetes documentation). It also allows the customer to control their update cycle, rather than leave it up to our release schedule.

The `kubelet` will only ever check for a new image version when a container is launched in any case, so using `:latest` in Kubernetes doesn't predictably guarantee the actual latest version unless some other automation is checking for a change to the image digest and triggering a new rollout. At best, we would need to instruct customers to scale the Pod to update, so explicit versioning is preferred.

Reply:
I see. Thank you for the explanation 🙂

Reply:
Of course! Thanks for reviewing this work. ❤️